FreeRTOS/FreeRTOS-Plus-TCP
# Fixed failing test cases for ARP unit tests

Description
-----------
This PR fixes the failing test cases for ARP unit tests which were failing after the IPv6 integration changes. 

Test Steps
-----------
```
cmake -S test/unit-test -B test/unit-test/build/ 
make -C test/unit-test/build/ all 
cd test/unit-test/build/
ctest -E system --output-on-failure --verbose
```

Related Issue
-----------
ARP unit test cases were failing due to the changes in the source code as part of the IPv6 integration

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
